packages/kbn-test/src/functional_test_runner/functional_test_runner.ts

```diff
@@ -74,7 +76,26 @@ export class FunctionalTestRunner {
         return (await providers.invokeProviderFn(customTestRunner)) || 0;
       }
 
-      const mocha = await setupMocha(this.lifecycle, this.log, config, providers, this.esVersion);
+      let reporter;
+      let reporterOptions;
+      if (config.get('mochaOpts.dryRun', false)) {
```

Comment:
The `false` default is defined in the schema, so I think it's a little redundant and distracting to put here

Reply:
Removed in 16d3e64